src/components/Checkbox/Checkbox.jsx

```diff
@@ -32,6 +33,8 @@ class Checkbox extends React.Component<Props> {
     value: false,
   };
 
+  id = this.props.id || uniqueId('id');
```

Comment:
what's the reason to derive it from props?

Reply:
in case someone wants to pass in an id (as people usually do... regular inputs support ids)